src/containers/monitor.jsx

```diff
@@ -135,6 +142,23 @@ class Monitor extends React.Component {
             mode: 'slider'
         }));
     }
+    handleSliderPromptClose () {
+        this.setState({sliderPrompt: false});
+    }
+    handleSliderPromptOpen () {
+        this.setState({sliderPrompt: true});
+    }
+    handleSliderPromptOk (min, max, decimal) {
+        if (min < max) {
```

Comment:
This was not the behavior in scratch2. In scratch 2 it seems like it just rearranged the min/max to make it work with what you typed in.